doc/manual/src/installation/uninstall.md

```diff
@@ -1,12 +1,52 @@
 # Uninstalling Nix
 
+## Sections
+- [Single User](#single-user)
+  - [Manual Method](#manual-method)
+    - [Step 1: Restore Modified Files (if applicable)](#step-2-restore-modified-files-if-applicable)
+    - [Step 2: Delete Nix Files](#step-4-delete-nix-files)
+- [Multi User](#multi-user)
+    - [Linux](#linux)
+    - [macOS](#macos)
+
+
 ## Single User
+### If you have a [single-user installation](./installing-binary.md#single-user-installation) of Nix
+### Manual Method
 
-If you have a [single-user installation](./installing-binary.md#single-user-installation) of Nix, uninstall it by running:
+#### Step 1: Restore Modified Files (if applicable)
+
+1. Nix may have modified certain files on your system. If you have created backup copies of these files before installing Nix, you can restore them. Here are the instructions for bash and zsh files:
```

Comment:
It's not perfectly correct to call them configuration files. Maybe @roberth has a better suggestion.

```suggestion
Nix will create backup copies of shell configuration files before installation, an you can restore them.
Here are the instructions for Bash and Zsh:
```

There is no second item in that list, so don't make it a list. Use one line per sentence, it's easier to review and make focused suggestions. This is the general rule here, even if it's not consistently implemented yet. Use the project names with their original spelling (Bash, Zsh). Use monospace for command names (`bash`, `zsh`, but that's not applicable here). I will add a contribution guide for the manual this year, promised.